public/styles/_releases.scss

```diff
@@ -117,7 +133,7 @@
 @media (min-width: 700px) {
```

Comment:
Can we use Primer breakpoint variables to standardize things?

Reply:
I updated that to primer min-width, but looks like them doesn't have breakpoints for `@media`